install_config/topics/ocp_osp_provisioning.adoc

```diff
@@ -189,8 +189,19 @@ openshift_node_open_ports:
 # set in the inventory file as openshift_openstack_kuryr_pod_subnet_cidr.
 # By default, this value is /24.
 
-# The value of openshift_portal_net in OSEv3.yml must also match the following value:
-# openshift_openstack_kuryr_service_subnet_cidr: *"172.23.192.0/18"*
+# The value of openshift_portal_net in OSEv3.yml must be within the range given
+# for openshift_openstack_kuryr_service_subnet_cidr (usually half of it),
```

Comment:
@luis5tb Why half, and when not half?

Reply:
explanation is just below. Octavia Load Balancer require an IP (besides the VIP), so for each k8s svc, we need 2 neutron ports (1 for the amphora VM, 1 for the loadbalancer VIP, which is the same as the one the svc gets)